include/zephyr/net/socket_ncs.h

```diff
@@ -25,33 +25,34 @@ extern "C" {
 #define SOCK_NATIVE 0x80000000
 #define SOCK_NATIVE_TLS 0x40000000
 
+/** Define a base for NCS specific socket options to prevent overlaps with Zephyr's socket options.
+ *  Allocates to NCS specific socket options the signed integer range 1073741824 to INT32_MAX.
+ */
+#define NET_SOCKET_NCS_BASE 0x4000000
```

Comment:
Could we use a round integer number here? It is not very useful to have a round Hex number in this case

Reply:
This number is exactly halfway through the positive integer range. Do you have a different suggestion?

Reply:
Yes, to use a round integer number like e.g. `100000`. The reason is that this number is `1073741824` in decimal, and combined with decimal definitions of socket option values makes it is hard to debug this values when we need to. For example, which socket option is `1073741842`?

When you print it in hex, it's `0x40000012` and note that `12` is in hex, so we need to convert to integer to figure out it's actually socket option `18` that is, `TLS_DTLS_HANDSHAKE_TIMEO`. If we saw `100018` it would be easier to tell what we are looking at.

Reply:
Thanks for the clarification.